crates/transaction-pool/src/validate/mod.rs

```diff
@@ -30,7 +30,8 @@ pub use constants::{
 
 /// A Result type returned after checking a transaction's validity.
 #[derive(Debug)]
-pub enum TransactionValidationOutcome<T: PoolTransaction> {
+pub enum TransactionValidationOutcome<T: PoolTransaction<Consensus = TransactionSignedEcRecovered>>
```

Comment:
I think we can keep it as just `T: PoolTransaction` in definition and some of the impls as only a few of them actually rely on conversions

Reply:
oups yeah fixed